aptos-move/aptos-e2e-comparison-testing/src/main.rs

```diff
@@ -35,6 +36,29 @@ pub enum Cmd {
         /// Dump the write set of txns
         #[clap(long, default_value_t = false)]
         dump_write_set: bool,
+        /// Target account
+        #[clap(long)]
+        target_account: Option<AccountAddress>,
+    },
+    /// Online execution
+    Online {
```

Comment:
Hmm I wasn't quite sure what this command means. Can you add more description on what this would do?

Reply:
Sorry for the confusion. With this command, the tool does not need to dump the state. Instead, it will try to get each txn, use DebuggerStateView the get the state, except for the code, which is obtained from compilation of the source code and load it to the code state. The logic is defined in data_stateview.rs in this PR.

Reply:
I think your explanation can go to the comment instead of `/// Online execution`

Reply:
done